mods/cnc/rules/aircraft.yaml

```diff
@@ -26,7 +26,8 @@ TRAN:
 	Encyclopedia:
 		Description: actor-tran.encyclopedia
 		Order: 230
-		Category: Units
+		Scale: 3
```

Comment:
I think scale of 3 does not look good - the art appears blurry:
![image](https://github.com/user-attachments/assets/34388c87-5e2c-484e-a478-044c73489b88)

A scale of 2 works much better (both for infantry and vehicles):
![image](https://github.com/user-attachments/assets/49ddb164-867f-4ad3-bff5-eb6c4347866d)

IMO there's nothing to gain from the 3 scaling factor.

And for structures while scale 2 looks OK I think scale 1 looks even better - I just like the crisp pixels.
![image](https://github.com/user-attachments/assets/424b3341-8a61-4edf-87e3-4a686d9dc06c)

Reply:
Strange, these should be using the crisp scaling filter so should not look blurry (and don't on my machine).

Reply:
Yeah, weird. Maybe it's my monitor, or my perception :eyes: Does my screenshot look blurry?

Reply:
even with crisp scaling they look a bit uncanny

Reply:
This may be a hidpi vs lowdpi issue - what screen resolution and UI scale are you using @dragunoff?

Reply:
Ok, it turns out that this is caused by a bug in the upscaling filter not accounting for the thing being rendered itself being scaled.

Reply:
More specifically, it is this: https://github.com/OpenRA/OpenRA/pull/17424#discussion_r355189553. My solution at the time was to calculate `AntialiasPixelsPerTexel` at the window level, but that implicitly assumes that sprites are being rendered at 1:1 scale. Changing to `dFdx/`dFdy` will fix it properly.

Reply:
Fixed in #21634.